lib/bundler/stats/cli.rb

```diff
@@ -38,6 +38,20 @@ def show(target)
         end
       end
 
+      desc 'Versions TARGET', 'Prints the dependency tree for a single gem in your Gemfile'
```

Comment:
Can you correct the case on `Versions` here, since the actual command uses lowercase?